docs/specs/casing.yml

```diff
@@ -85,3 +85,37 @@ outputs:
   docs/A.json: |
     {"conceptual":"<p><a href=\"B\"></a></p>"}
   docs/B.json:
+---
+# Use real file casing in manifest
+inputs:
+  docfx.yml: |
+    files: '*'
+  a.md: '[](media/ImAGe.PNG) [](MEdia/ImAGe.PNG)'
```

Comment:
if we already have the casing file name, why we need lower case the resolved urls?

Reply:
For SEO purpose, output URLs are canonicalized into lower case because Google considers URLs to be case-sensitive

Reply:
that's a different behavior with v2 right? so we need first figure out what's the SEO purpose for v2 behavior.  Before this PR, I though we lower case all resolve urls is a compromise way

Reply:
v2 also lower case urls, search `ARIA-and-UI-Automation` in [source](https://github.com/MicrosoftDocs/edge-developer/edit/master/microsoft-edge/accessibility/build.md) and [result](https://review.docs.microsoft.com/en-us/microsoft-edge/accessibility/build)